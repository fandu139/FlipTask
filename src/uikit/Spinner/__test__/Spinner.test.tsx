import Spinner from '../../Spinner';
import React from 'react';
import { render } from '@testing-library/react-native';

describe('Spinner component test', () => {
  describe('snapshot test', () => {
    it('should render correctly', () => {
      const tree = render(<Spinner />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
