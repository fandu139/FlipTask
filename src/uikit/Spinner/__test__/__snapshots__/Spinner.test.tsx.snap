// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component test snapshot test should render correctly 1`] = `
<ActivityIndicator
  accessibilityLabel="loading-indicator"
  animating={true}
  color="#EF4941"
  hidesWhenStopped={true}
  size="large"
  testID="loading-indicator"
/>
`;
